README.md

```diff
@@ -55,4 +60,6 @@ WebGoat (48K LOC)
 WebGoat7 (106K LOC)
 ```
 
+## Building
 
+Use `./mvn verify` to build and test changes to the project

```

Comment:
`./mvnw verify`

Reply:
consider `./mvnw verify clean install` instead so that the developer remembers to install.

Reply:
Are you sure users need to `install`? If they're just trying to test their changes to this project, it should be unnecessary. The Maven community has been trying to re-educate users on when they actually need to `clean` and `install`, and their message is that most of the time you just need `mvn verify` 

* https://twitter.com/ASFMavenProject/status/1200844862897172480
* https://twitter.com/ASFMavenProject/status/1002840059337412608

Reply:
yeah. `verify` is probably sufficient.

Reply:
Cool. @bryanateoan are you still waiting on me to make changes somewhere? Your review is still in "Changes Requested" state

Reply:
can you change the `./mvn verify` to `./mvnw verify`?

Reply:
Ah I didn't see that change. Pushed